css/media.css

```diff
@@ -0,0 +1,319 @@
+@media(min-width: 320px) {
+    body {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)